controller/deployments.go

```diff
@@ -481,90 +454,6 @@ func (c *DeploymentsController) ShowSpace(ctx *app.ShowSpaceDeploymentsContext)
 	return ctx.OK(res)
 }
 
-// ShowSpaceApp runs the showSpaceApp action.
-func (c *DeploymentsController) ShowSpaceApp(ctx *app.ShowSpaceAppDeploymentsContext) error {
```

Comment:
If any API from this PR might be required in the future, it's probably going to be /api/spaces/XXX/applications/YYY.  Are you sure this should go?